pkg/controller/verticalpodautoscaler/verticalpodautoscaler_controller.go

```diff
@@ -500,6 +702,100 @@ func (r *Reconciler) VPAPodSpec(vpa *autoscalingv1.VerticalPodAutoscalerControll
 	return spec
 }
 
+// AdmissionControllerPodSpec returns the expected podSpec for the Admission Controller deployment belonging
+// to the given VerticalPodAutoscalerController.
+func (r *Reconciler) AdmissionControllerPodSpec(vpa *autoscalingv1.VerticalPodAutoscalerController, params ControllerParams) *corev1.PodSpec {
+	spec := r.VPAPodSpec(vpa, params)
+	spec.Containers[0].Env = append(spec.Containers[0].Env, corev1.EnvVar{
+		Name: "NAMESPACE",
+		ValueFrom: &corev1.EnvVarSource{
+			FieldRef: &corev1.ObjectFieldSelector{
+				FieldPath: "metadata.namespace",
+			},
+		},
+	})
+	spec.Containers[0].Ports = append(spec.Containers[0].Ports, corev1.ContainerPort{
+		ContainerPort: 8000,
```

Comment:
Is this port configurable?

Reply:
It is configurable in the admission controller, but it doesn't need to be configurable by the end user. I think it's okay for us to be opinionated about it, since the only thing connecting to it will be the API server and it will use the service that we create. So the port number doesn't really matter. It might make sense for us to make it a constant though so that it stays in sync. What do you think?

Reply:
sounds good